### PR TITLE
Feature/add review frontend component

### PR DESCRIPTION
**Pull request by aLLeXUs**

### CODE REVIEW

File: app/Actions/Review/SaveReviewPhotoAction.php [hunk 1]
```diff
@@ -25,7 +25,7 @@ public function execute(SaveReviewPhotoRequest $request): SaveReviewPhotoRespons
             $reviewPhoto = $this->reviewPhotoRepository->getById($id);
         }
         $file = $request->getImg();
-        $newFileName = time() . '.' . $file->extension();
+        $newFileName = time() . '_' . mt_rand() . '.' . $file->extension();
```
Comment (nemoipaha) [inline app/Actions/Review/SaveReviewPhotoAction.php]: Why we have mt_rand ?

Comment (aLLeXUs) [reply app/Actions/Review/SaveReviewPhotoAction.php]: when upload more than one photo, it can be uploaded with same name, and will be overwrited

Comment (nemoipaha) [reply app/Actions/Review/SaveReviewPhotoAction.php]: @aLLeXUs But photos upload works in async mode, so time() should be enough. Also would be nice to have separate class like FileNameGenerator to create keys like `time() . '_' . mt_rand() . '.' . $file->extension();`, and accept $file as constructor param

File: frontend/src/components/review/AddReview.vue [hunk 2]
```diff
@@ -0,0 +1,158 @@
+<template>
+    <div>
+        <div v-if="hasToken" class="add-review">
+            <b-field class="media">
+
+                <div class="media-left">
+                    <img
+                        src="https://ss0.4sqi.net/img/venuepage/v2/add_tip_blank_avatar@2x-4321684c656168f26ae9208901a9d83e.png"
```
Comment (lenchvolodymyr) [inline frontend/src/components/review/AddReview.vue]: download this placeholder and store it in the assets directory

File: frontend/src/components/review/AddReview.vue [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+                        class="button is-primary"
+                        @click="onAddReview"
+                    >Post</button>
+                </div>
+            </b-field>
+        </div>
+        <div v-else class="add-review media">
+            <div class="media-left">
+                <img
+                    src="https://ss0.4sqi.net/img/venuepage/v2/add_tip_blank_avatar@2x-4321684c656168f26ae9208901a9d83e.png"
```
Comment (lenchvolodymyr) [inline frontend/src/components/review/AddReview.vue]: the same

File: frontend/src/components/review/AddReview.vue [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+                    disabled
+                >Post</button>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import { mapActions } from 'vuex';
```
Comment (lenchvolodymyr) [inline frontend/src/components/review/AddReview.vue]: you can use one import construction

`import { mapGetters, mapActions } from 'vuex';`

**Pull request merged**